packages/graph-explorer/src/modules/NodeExpand/NodeExpandFilters.tsx

```diff
@@ -12,6 +12,7 @@ import {
 import { useConfiguration } from "../../core";
 import useTextTransform from "../../hooks/useTextTransform";
 import useTranslations from "../../hooks/useTranslations";
+import { useVertexTypeConfig } from "../../core/ConfigurationProvider/useConfiguration";
```

Comment:
is this replacement for `useConfiguration`?

Reply:
Sort of. My overall long term goal is to break apart `useConfiguration` in to smaller elements.

I'm also trying to move more state projections in to recoil and end up with less callbacks that simply return projected state in the react code.

Reply:
I see that in some cases used 
`const config = useConfiguration();
const vConfig = config?.getVertexTypeConfig(vt);`
but in other
`const vtConfig = useVertexTypeConfig(selectedType);`

will this be changed in the next PR's?

Reply:
Over time, yes. The ones that remain fall in to two camps:

- **Non-React Code** - Things like the list item or table callbacks for data to fill their items
- **Targeted Info** - Some components want the display name or total specifically and I want to handle these cases differently